examples/stream.ru

```diff
@@ -3,7 +3,6 @@
 # run *one* of these:
 #
 #   rackup -s mongrel stream.ru   # gem install mongrel
-#   thin -R stream.ru start       # gem install thin
```

Comment:
Please copy `test/integration/rainbows.conf` here and add `rainbows -c rainbows.conf stream.ru`.

Reply:
fixed 7a96472